helm/charts/kratos-selfservice-ui-node/README.md

```diff
@@ -6,30 +6,31 @@ A Helm chart for ORY Kratos's example ui for Kubernetes
 
 ## Values
 
-| Key | Type | Default | Description |
-|-----|------|---------|-------------|
-| affinity | object | `{}` |  |
-| baseUrl | string | `""` | The baseUrl |
-| deployment | object | `{"annotations":{},"automountServiceAccountToken":false,"extraEnv":[],"extraVolumeMounts":[],"extraVolumes":[],"labels":{},"nodeSelector":{},"resources":{},"tolerations":[]}` | Deployment configuration |
-| deployment.extraEnv | list | `[]` | Array of extra envs to be passed to the deployment. Kubernetes format is expected - name: FOO   value: BAR |
-| deployment.extraVolumes | list | `[]` | If you want to mount external volume For example, mount a secret containing Certificate root CA to verify database TLS connection. |
-| deployment.nodeSelector | object | `{}` | Node labels for pod assignment. |
-| deployment.tolerations | list | `[]` | Configure node tolerations. |
-| fullnameOverride | string | `""` |  |
-| image | object | `{"pullPolicy":"IfNotPresent","repository":"oryd/kratos-selfservice-ui-node","tag":"v0.8.0-alpha.3"}` | Deployment image settings |
-| image.tag | string | `"v0.8.0-alpha.3"` | ORY KRATOS VERSION |
-| imagePullSecrets | list | `[]` |  |
-| ingress | object | `{"annotations":{},"className":"","enabled":false,"hosts":[{"host":"chart-example.local","paths":[{"path":"/","pathType":"ImplementationSpecific"}]}],"tls":[]}` | Ingress configration |
-| jwksUrl | string | `"http://oathkeeper-api"` | The jwksUrl |
-| kratosAdminUrl | string | `"http://kratos-admin"` | Set this to ORY Kratos's Admin URL |
-| kratosBrowserUrl | string | `"http://kratos-browserui"` | Set this to ORY Kratos's public URL accessible from the outside world. |
-| kratosPublicUrl | string | `"http://kratos-public"` | Set this to ORY Kratos's public URL |
-| nameOverride | string | `""` |  |
-| projectName | string | `"SecureApp"` |  |
-| replicaCount | int | `1` | Number of replicas in deployment |
-| securityContext | object | `{"allowPrivilegeEscalation":false,"capabilities":{"drop":["ALL"]},"privileged":false,"readOnlyRootFilesystem":true,"runAsNonRoot":true,"runAsUser":1000}` | Deployment level securityContext |
-| service | object | `{"name":"http","port":80,"type":"ClusterIP"}` | Service configuration |
-| service.name | string | `"http"` | The service port name. Useful to set a custom service port name if it must follow a scheme (e.g. Istio) |
+| Key                     | Type   | Default                                                                                                                                                                        | Description                                                                                                                        |
```

Comment:
Those files are auto-generated by helm-docs plugin from our values.yaml. The `make format` and the plugin will override each other all the time 😞